lib/register.js

```diff
@@ -1,26 +1,4 @@
 'use strict';
```

Comment:
this file can probably be deleted, and a global registry be created directly in `registry.js`